testing/main_integration_test.go

```diff
@@ -128,7 +128,7 @@ type Package struct {
 }
 
 func getPackages(t *testing.T) ([]string, error) {
-	resp, err := http.Get("http://localhost:8080/search")
+	resp, err := http.Get("http://localhost:8080/search?experimental=true")
```

Comment:
@ph @jonathan-buttner Looking at the diff between staging / production, this is what I found.